dotcom-rendering/src/components/EditionSwitcherBanner.importable.tsx

```diff
@@ -29,6 +29,11 @@ const content = css`
 	align-items: flex-start;
 	${center}
 
+	svg {
+		fill: ${palette.brand[400]};
+		margin-right: ${space[3]}px;
```

Comment:
Could we use `gap` in the `textAndLink` styles to apply the horizontal spacing? (It was previously being used anyway.) Together with the `theme` prop that avoids having to override icon styles.